src/pkg/packager/prepare.go

```diff
@@ -204,36 +205,38 @@ func (p *Packager) FindImages(baseDir, repoHelmChartPath string) error {
 
 		if sortedImages := k8s.SortImages(matchedImages, nil); len(sortedImages) > 0 {
 			// Log the header comment
-			fmt.Printf("\n  - name: %s\n    images:\n", component.Name)
+			componentDefinition += fmt.Sprintf("\n  - name: %s\n    images:\n", component.Name)
 			for _, image := range sortedImages {
 				// Use print because we want this dumped to stdout
-				fmt.Println("      - " + image)
+				componentDefinition += fmt.Sprintf("      - %s\n", image)
 			}
 		}
 
 		// Handle the "maybes"
 		if sortedImages := k8s.SortImages(maybeImages, matchedImages); len(sortedImages) > 0 {
-			var realImages []string
+			var validImages []string
 			for _, image := range sortedImages {
 				if descriptor, err := crane.Head(image, config.GetCraneOptions(config.CommonOptions.Insecure)...); err != nil {
 					// Test if this is a real image, if not just quiet log to debug, this is normal
 					message.Debugf("Suspected image does not appear to be valid: %#v", err)
 				} else {
 					// Otherwise, add to the list of images
 					message.Debugf("Imaged digest found: %s", descriptor.Digest)
-					realImages = append(realImages, image)
+					validImages = append(validImages, image)
 				}
 			}
 
-			if len(realImages) > 0 {
-				fmt.Printf("      # Possible images - %s - %s\n", p.cfg.Pkg.Metadata.Name, component.Name)
-				for _, image := range realImages {
-					fmt.Println("      - " + image)
+			if len(validImages) > 0 {
+				componentDefinition += fmt.Sprintf("      # Possible images - %s - %s\n", p.cfg.Pkg.Metadata.Name, component.Name)
+				for _, image := range validImages {
+					componentDefinition += fmt.Sprintf("      - %s\n", image)
 				}
 			}
 		}
 	}
 
+	fmt.Println(componentDefinition)
```

Comment:
Do we want to create a `message.Text` so that any STDERR/STDOUT redirections from pterm are used globally?

Reply:
I think it is easy enough to use fmt.Println to print things to stdOut - I like that it is very explicit when we are printing to std out vs std err.  One thing I suppose though is that it doesn't currently print that output in the logs - though we only use this in more prepare or tools like scenarios.